Question/Question.html

```diff
@@ -0,0 +1,68 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Blithe Quiz</title>
+    <link rel="stylesheet" href="./Question.css">
+</head>
+<body>
+
+    <!-- header -->
+    <div class="header-container">
+        <header class="header">
+            <h2 class="header-title">
+                Blithe Quiz App
+            </h2>
+            <div class="header-icon-container">
+                <div class="header-icon-sub-container">
+                    <button class="header-btn">
+                        <img src="https://img.icons8.com/material-outlined/24/000000/share.png"
+                            class="icon-btn-img" alt="icon-btn-img" />
+                    </button>
+                    <div class="badge-container">
+                        <img class="avatar xs"
+                            src="https://image.freepik.com/free-vector/businessman-character-avatar-isolated_24877-60111.jpg"
+                            alt="user">
+                    </div>
+                </div>
+            </div>
+        </header>
+    </div>
+
+    <div class='col-12 row quiz-home-box'>
+        <p class='heading-md'>Covid Essentials</p>
+        <div class="col-6 stats">
+			<div class="quiz-time">
+				<div class="text-md remianing-time">00</div>
+				<span class="text-md time-up-text"><b>Time's Up !!</b></span>
+			</div>
+			<div class="score-board">
+				<span class="text-md score-text">Score:</span>
+				<span class="text-md correct-answers">10</span>
+			</div>
+		</div>
+        <!-- <div class="que-container"> -->
+            <div class="col-6 question-box">
+                <div class="current-que-num">1/20</div>
+                <div class="question-text">
+                    Do ur fingernails and hairs grow after you die ?
+                </div>
+            </div>
+            <div class="col-6 option-box">
+                <div class="option correct">option 1</div>
+                <div class="option">option 2</div>
+                <div class="option">option 3</div>
+                <div class="option wrong">option 4</div>
+            </div>
+            <div class=" col-6 answer-description">
+                They really don'tvdfnvkdfnjkvnfdjknvjkfdnjkvnjfdknvjkdfnsjkvnjkfdnjkvnjkdsfnjkvnsjkdfnvjknsdfjkvjksdnfjkvsjkdfnjkvnjksdfnvjkndfj vsdhgvcghsdvghcvsdghvcghvagsdcghvdsghjcvghdsvghcvghadscghvdsghcvghdsvghcvghdsvcghavghsdhgv
+            </div>
+    
+            <div class="col-6 next-question">
+                <button type="button" class="btn btn-sm btn-primary btn-next-que">Next Question</button>
+                <a href="../Categories/Categories.html" class="btn text-decoration-none btn-sm btn-link btn-see-result">See Your Result !</a> 
```

Comment:
Small UI suggestion, usually 'next question' button is on the right side of the page. Other than this you are good to merge